### PR TITLE
Use DI to inject environment variables

### PR DESCRIPTION
**Pull request by mbryzek**
- Do NOT require google api key in test
    - Validate env vars in healthcheck

### CODE REVIEW

File: api/app/controllers/Healthchecks.scala [hunk 1]
```diff
@@ -5,7 +5,10 @@ import io.flow.common.v0.models.json._
 import play.api.mvc._
 import play.api.libs.json._
 
-class Healthchecks extends Controller {
+@javax.inject.Singleton
+class Healthchecks @javax.inject.Inject() (
+  environmentVariables: utils.EnvironmentVariables
```
Comment (mikeclimbrock) [inline api/app/controllers/Healthchecks.scala]: putting here for future reference or was intention to use it this time around?

Comment (mikeclimbrock) [inline api/app/controllers/Healthchecks.scala]: disregard, i see it will make a difference as to which mode is used

**Comment by mikeclimbrock** [comment]
looks like rebase needed

**Comment by mbryzek** [comment]
rebased

Yeah: by injecting the env vars, given we use vals, they will all be resolved at healthcheck time ensuring they are defined

**Comment by mikeclimbrock** [comment]
lgtm

**Pull request merged**